### DISCUSSION TITLE
Add an local captcha module

### DISCUSSION THREAD

**Discussion by fMichaleczek** [OrchardCMS/OrchardCore#7457] [Ideas]
It could be nice to provide a captcha module  
 without any external server dependencies on closed-source.

It seems that FriendlyCaptcha is the only one open source projet which can be hosted by yourself.
https://docs.friendlycaptcha.com/

Do you know others solutions ? 
Do you think it could add a great value to Orchard ?

**Comment by Skrypt** [comment]
Please look at this PR which allows extending Captcha : 
https://github.com/OrchardCMS/OrchardCore/pull/5932
I did not have time to review it properly but it looked fine on first look.
